crates/e2e/src/setup/onchain_components/safe.rs

```diff
@@ -0,0 +1,297 @@
+use {
+    super::{OnchainComponents, TestAccount},
+    contracts::{
+        GnosisSafe,
+        GnosisSafeCompatibilityFallbackHandler,
+        GnosisSafeProxy,
+        GnosisSafeProxyFactory,
+    },
+    ethcontract::{Bytes, H160, H256, U256},
+    hex_literal::hex,
+    model::{
+        order::OrderCreation,
+        signature::{hashed_eip712_message, Signature},
+        DomainSeparator,
+    },
+    shared::ethrpc::Web3,
+    web3::{
+        signing,
+        signing::{Key, SecretKeyRef},
+    },
+};
+
+//#[macro_export]
```

Comment:
I guess the `macro_export` can be dropped then?

Reply:
If you don't want to `macro_export`, I _think_ you can add (after the macro definition) a:

```rust
pub(crate) use self::tx_safe;
```